app/models/order.rb

```diff
@@ -19,6 +19,14 @@ def self.any_complete?
     where(status: "complete").count > 0
   end
 
+  def self.cancel(order_id)
+    order = self.find(order_id)
+    order.status = "cancelled"
+    order.save
+    OrderItem.return_inventory_for(order.id)
+    OrderItem.unfulfill_items_for(order.id)
+  end
```

Comment:
great refactor for the cancel method!  ❤️ fat models!

Reply:
+1